### ISSUE TITLE
TypeError: Object of type 'ndarray' is not JSON serializable

### ISSUE THREAD

**Issue by Onandon11** [PaulSchweizer/flowpipe#63]
When using numpy array as outputs there is a problem in with jsonify the numpy array. 

Code to reproduce (Python 3.6.5):
```
from flowpipe.node import Node
from flowpipe.graph import Graph

import numpy as np

@Node(outputs=['xyz_out'])
def getData(file):
    return {'xyz_out': np.ones((100,3))}

@Node(outputs=['xyz_mean'])
def getMean(xyz):
    return {'xyz_mean': xyz.mean(axis=0)}

@Node(outputs=['xyz_out'])
def substractMean(xyz, mean):
    return {'xyz_out': xyz - mean}

@Node()
def printSummary(xyz):
    print(xyz.mean(axis=0))

# Create a graph with the necessary nodes
graph = Graph(name='Data processing')
data = getData(graph=graph, name='Get XYZ data', file='filename')
mean = getMean(graph=graph, name='Get mean of XYZ data')
data_sub = substractMean(graph=graph, name='Set data to 0,0,0')
printer = printSummary(graph=graph, name='I\'m a pretty printer')

# Wire up the connections between the nodes
data.outputs['xyz_out']     >> mean.inputs['xyz']
data.outputs['xyz_out']     >> data_sub.inputs['xyz']
mean.outputs['xyz_mean']    >> data_sub.inputs['mean']
data_sub.outputs['xyz_out'] >> printer.inputs['xyz']


print(graph.name)
print(graph)
print(graph.list_repr())

import logging
from flowpipe import logger

logger.setLevel(logging.INFO)

graph.evaluate(threaded=False)
```

**Comment by PaulSchweizer** [comment]
Thanks for the report @Onandon11 I'll take a look at this later today.

**Comment by PaulSchweizer** [comment]
I tried your test code with the latest code from master and it worked. I think @neuneck had already identified this issue and fixed it but we just hadn't release it yet. 
I released it now as v0.5.1 (https://pypi.org/project/flowpipe/) please try it again and let me know if this issue is fixed for you as well now

**Comment by Onandon11** [comment]
Hi @PaulSchweizer,

Indeed you are right. I downloaded the code using pip and checked for the version (0.5.0) so I assumed that I had the latest source code. But a diff showed me otherwise, I downloaded the code from Github and now it's working indeed.
